onnx/optimizer/passes/extract_constant_to_initializer.h

```diff
@@ -0,0 +1,51 @@
+// ATTENTION: The code in this file is highly EXPERIMENTAL.
+// Adventurous users should note that the APIs will probably change.
+
+#pragma once
+
+#include "onnx/optimizer/passes/optimize_pass.h"
+
+namespace ONNX_NAMESPACE { namespace optimization {
+
+    struct ExtractConstantToInitializer final : public OptimizePass{
+
+        explicit ExtractConstantToInitializer()
+            : OptimizePass("extract_constant_to_initializer", API_TYPE::IR){
+            }
+
+        void extract_constant_to_initializer(Graph& graph){
+            for (auto it = graph.begin(); it != graph.end(); ++it){
+                auto* n = *it;
+                DescendOnGraphAttributes(n, [this](Graph& g){extract_constant_to_initializer(g);});
+                if (n->kind() == kConstant){
+                    
+                    const auto name = n->output()->uniqueName();
+                    
+                    Symbol sym = Symbol("value");
```

Comment:
It's interned - I guess you can just use `kValue`

Reply:
I guess you mean kvalue?